app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,12 @@
+class GuestsController < ApplicationController
+
+  def show
+    @guest = Guest.find(params[:id])
+    @rooms = @guest.rooms
+  end
+
+  def create
+    room_guest = RoomGuest.create!(room_id: params[:room_id], guest_id: params[:id] )
```

Comment:
The functionality (the "meat") of this method is fine; I don't think it necessarily belongs inside the `GuestsController`.... Consider if we needed to create a new `Guest` object; where would this go? 

When you're trying to figure out where to put a specific controller action, think about what is being done in that controller action first - in here, we're creating a new `RoomGuest`. We're not creating a new `Guest`, just using an ID to make a new _relationship_ between that Guest and a specific Room. So, we can make a new controller called `room_guests_controller` and put this `create` action in there.